coredns/tox.ini

```diff
@@ -2,7 +2,7 @@
 minversion = 2.0
 basepython = py37
 envlist =
-    py{27,37}-{1.2.0}
+    py{27,37}-{1.2.0}: e2e ready 
```

Comment:
This should be in `description`otherwise I get
```
ddev env ls coredns
Traceback (most recent call last):
  File "/Users/julia.simon/src/integrations-core/venv/bin/tox", line 10, in <module>
    sys.exit(cmdline())
  File "/Users/julia.simon/src/integrations-core/venv/lib/python3.7/site-packages/tox/session/__init__.py", line 44, in cmdline
    main(args)
  File "/Users/julia.simon/src/integrations-core/venv/lib/python3.7/site-packages/tox/session/__init__.py", line 64, in main
    config = load_config(args)
  File "/Users/julia.simon/src/integrations-core/venv/lib/python3.7/site-packages/tox/session/__init__.py", line 80, in load_config
    config = parseconfig(args)
  File "/Users/julia.simon/src/integrations-core/venv/lib/python3.7/site-packages/tox/config/__init__.py", line 262, in parseconfig
    ParseIni(config, config_file, content)
  File "/Users/julia.simon/src/integrations-core/venv/lib/python3.7/site-packages/tox/config/__init__.py", line 1081, in __init__
    res = self._getenvdata(reader, config)
  File "/Users/julia.simon/src/integrations-core/venv/lib/python3.7/site-packages/tox/config/__init__.py", line 1270, in _getenvdata
    from_config = reader.getstring("envlist", replace=False)
  File "/Users/julia.simon/src/integrations-core/venv/lib/python3.7/site-packages/tox/config/__init__.py", line 1482, in getstring
    x = self._apply_factors(x)
  File "/Users/julia.simon/src/integrations-core/venv/lib/python3.7/site-packages/tox/config/__init__.py", line 1508, in _apply_factors
    return "\n".join(filter(None, map(factor_line, lines)))
  File "/Users/julia.simon/src/integrations-core/venv/lib/python3.7/site-packages/tox/config/__init__.py", line 1503, in factor_line
    for included, excluded in _split_factor_expr(expr)
  File "/Users/julia.simon/src/integrations-core/venv/lib/python3.7/site-packages/tox/config/__init__.py", line 1503, in <genexpr>
    for included, excluded in _split_factor_expr(expr)
TypeError: '<=' not supported between instances of 'set' and 'tuple'
```